site/assignments/faq.rst

```diff
@@ -0,0 +1,91 @@
+***************************************************
+General Assignment Frequently Asked Questions (FAQ)
+***************************************************
+
+* I don't know how to do *X*
+
+    * This is typical and sometimes intentional
+    * Go to `google.ca <https://www.google.ca>`_ and type in *X*
+    * This is not to dismiss your question --- it's how experienced programmers figure out how to do *X*
+
+
+* How long should this assignment take?
+
+    * This will depend heavily on experience levels
+    * At this stage, expect the assignments to take multiple hours
+    * In other words, do not get discouraged if you are on hour 4 and you're yet to finish
+    * Remember, you would not expect to write a high quality essay in a single one hour sitting
+
+
+* I think I did everything correctly, but it's still not working
+
+    * This is typical
+    * Sometimes bugs are simple and easy to find, and others are sneaky
+    * Take your time with debugging and tracking down the problem
+    * I recommend going through the code; what does the code you wrote actually say?
```

Comment:
probably worth suggesting typical approaches, like adding print statements to display variable values, or cracking open the debugger if they are so inclined